gzll/doc/integration_notes.rst

```diff
@@ -7,12 +7,28 @@ Integration notes
    :local:
    :depth: 2
 
+To integrate the Gazell Link Layer library in your application, you need to:
+
+* Link in the GZLL library. See the :ref:`gzll_configuration` section.
+* Supply the glue code used by the library. See the :ref:`gzll_glue_layer` section.
+
 RTOS
 ****
 
 Gazell Link Layer API is not reentrant.
 It should be called by only a single thread in an RTOS.
 
+.. _gzll_configuration:
+
+Configuration
+*************
+
+In the |NCS|, you can enable the GZLL library using the :kconfig:`CONFIG_GZLL` Kconfig option.
+Look for the menu item "Enable Gazell Link Layer".
```

Comment:
Where do you look for this menu item? In SES or VS Code?

Reply:
The Kconfig can be organized into a menu. In the build system, there are options 'menuconfig' and 'guiconfig'. SES and VS Code can also view the menu.

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/build/index.html#configuration-system-kconfig